runtime/gen_primitives.sh

```diff
@@ -28,9 +28,20 @@ export LC_ALL=C
       dynlink backtrace_byt backtrace afl \
       bigarray prng
   do
-      sed -n -e 's/^CAMLprim value \([a-z0-9_][a-z0-9_]*\).*/\1/p' \
-        "runtime/$prim.c"
+      if [[ $1 = "RUNNING_FROM_RUNTIME_DIR" ]]; then
+          output="$prim.c"
+      else
+          output="runtime/$prim.c"
+      fi
```

Comment:
For what is worth, what I do to make scripts robust wrt to the current directory is to add a `cd $(dirname "$0")` at the top of the script, which has the effect of putting yourself in the directory that contains the script itself (here, `runtime/`).